mti_nma/bin/make_9grid_mode_video.py

```diff
@@ -0,0 +1,27 @@
+import imageio
```

Comment:
This is awesome! Excited to try it out.

Reply:
Let me know how it goes! Obviously its not part of the pipeline so you have to run this is python separately after you've gone into Paraview and made the image files. Happy to get feedback on how to make this process smoother or clearer fi you try it out.